src/Palettes.md

```diff
@@ -113,18 +130,18 @@ Even though GBA is described to be compatible to CGB games, most CGB
 games are completely unplayable on older GBAs because most colors are
 invisible (black). Of course, colors such like Black and White will
 appear the same on both CGB and GBA, but medium intensities are arranged
-completely different. Intensities in range 00h..07h are invisible/black
+completely different. Intensities in range $00..07 are invisible/black
```

Comment:
Don't we use `-` for ranges?

Reply:
`–`, to be precise, but yes. Should probably go in the [style guide](https://github.com/gbdev/pandocs/wiki/Document-Style).

Reply:
I agree